CLAUDE.md

```
@@ -0,0 +1,318 @@
+├── README.md
+└── .env.example
+```
+
+### Tech Stack
+```json
+{
+  "core": {
+    "language": "TypeScript",
+    "runtime": "Node.js 20+",
+    "package-manager": "pnpm"
+  },
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)